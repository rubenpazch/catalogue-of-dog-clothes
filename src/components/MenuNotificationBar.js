import React from 'react';
import { Col, Row } from 'react-bootstrap';
import styles from '../css/menunotificationbar.module.css';

function MenuNotificationBar() {
  return (
    <Row className={styles.wrapper} data-testid="MenuNotificationBar">
      <Col className={styles.leftwrapper} md={2} />
      <Col className={styles.midlewrapper} md={7}>
        <ul className={styles.wrapperMenu}>
          <li href="#home" className={styles.itemList}>
            <a href="localhost:3000">Most Populars Places</a>
          </li>
          <li href="#features" className={styles.itemList}>
            <a href="localhost:3000"> Last Time Trip</a>
          </li>
          <li href="#pricing" className={styles.itemList}>
            <a href="localhost:3000">Flexible Itinerary</a>
          </li>
          <li href="#pricing" className={styles.itemList}>
            <a href="localhost:3000">Deals</a>
          </li>
        </ul>
      </Col>
      <Col className={styles.rigthwrapper} md={3}>
        <p>-</p>
      </Col>
    </Row>
  );
}

export default MenuNotificationBar;
