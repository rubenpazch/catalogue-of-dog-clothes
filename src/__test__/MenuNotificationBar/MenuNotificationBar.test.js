import React from 'react';
import ReactDOM from 'react-dom';
import {
  BrowserRouter as Router,
  Route,
} from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import MenuNotificationBar from '../../components/MenuNotificationBar';

afterEach(cleanup);

it('renders MenuNavBar without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <Router>
      <Route>
        <MenuNotificationBar />
      </Route>
    </Router>,
    div,
  );
});

it('renders item correctly', () => {
  const { getByTestId } = render(
    <Router>
      <Route>
        <MenuNotificationBar />
      </Route>
    </Router>,
  );
  getByTestId('MenuNotificationBar');
});

it('render ItemTour information', () => {
  const { getByTestId } = render(
    <Router>
      <Route>
        <MenuNotificationBar />
      </Route>
    </Router>,
  );
  expect(getByTestId('MenuNotificationBar')).toHaveTextContent('Most Populars Places');
  expect(getByTestId('MenuNotificationBar')).toHaveTextContent('Last Time Trip');
  expect(getByTestId('MenuNotificationBar')).toHaveTextContent('Flexible Itinerary');
  expect(getByTestId('MenuNotificationBar')).toHaveTextContent('Deals');
});

it('machets snapshot MenuNavBar', () => {
  const itemTour = renderer.create(
    <Router>
      <Route>
        <MenuNotificationBar />
      </Route>
    </Router>,
  ).toJSON();
  expect(itemTour).toMatchSnapshot();
});
