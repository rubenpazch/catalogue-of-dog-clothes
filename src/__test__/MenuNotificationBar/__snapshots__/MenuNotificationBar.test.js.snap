// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`machets snapshot MenuNavBar 1`] = `
<div
  className="wrapper row"
  data-testid="MenuNotificationBar"
>
  <div
    className="leftwrapper col-md-2"
  />
  <div
    className="midlewrapper col-md-7"
  >
    <ul
      className="wrapperMenu"
    >
      <li
        className="itemList"
        href="#home"
      >
        <a
          href="localhost:3000"
        >
          Most Populars Places
        </a>
      </li>
      <li
        className="itemList"
        href="#features"
      >
        <a
          href="localhost:3000"
        >
           Last Time Trip
        </a>
      </li>
      <li
        className="itemList"
        href="#pricing"
      >
        <a
          href="localhost:3000"
        >
          Flexible Itinerary
        </a>
      </li>
      <li
        className="itemList"
        href="#pricing"
      >
        <a
          href="localhost:3000"
        >
          Deals
        </a>
      </li>
    </ul>
  </div>
  <div
    className="rigthwrapper col-md-3"
  >
    <p>
      -
    </p>
  </div>
</div>
`;
